generators/client-2/templates/src/main/webapp/app/blocks/interceptor/_http.interceptor.ts

```diff
@@ -31,7 +31,7 @@ export class HttpInterceptor extends Http {
     }
 
     request(url: string | Request, options?: RequestOptionsArgs): Observable<Response> {
-        return this.intercept(super.request(url, this.getRequestOptionArgs(options)));
+        return super.request(url, this.getRequestOptionArgs(options));
```

Comment:
In fact, I think it's intrinsic to the http class. Each time we try make an http request the appropriate method (get , put, post...) is called and then  the request method is called. So if we intercept it in every method (like before my commit) you intercept the request twice. The first one in the get method and then in the request method for exemple. You can try it by yourself if you add some logs in each method of the http.interceptor and in each interceptor to understand what l'm talking about. Sorry if l was unclear  !